sample_apps/rso_sample_apps/README.md

```diff
@@ -0,0 +1,72 @@
+# samples
+The goal of this repository is to show you how quickly you can start creating your own backend service that implements your Riot approved app.
+
+This repo contains samples of the most popular programming languages:
+
+* python 🐍
+* nodejs 🐢
+* golang 🦫
+
+# ⚠️ Warning ⚠️
+These samples are not suited for production environments, this samples are intended to show how RSO can be used to consume Riot API endpoints.
+
+# Requirements to run the samples ✅
+* Get your secrets:
+    * RSO_BASE_URL=https://auth.riotgames.com
+    * RSO_CLIENT_ID=<your-client-id>
+    * RSO_CLIENT_SECRET=<your-client-secret>
+    * DEPLOYMENT_API_KEY=<your-deployment-api-key>
+    * APP_BASE_URL=http://<your-app-base-url>
+    * APP_CALLBACK_PATH=<your-callback-path>
+    * RESPONSE_TYPE=code
+    * SCOPE=openid
+* Make sure you have your `.env` in the root of the project: `samples/.env` not `samples/.env`
+* Add `127.0.0.1       local.exampleapp.com` to your hosts file.
+    * MacOS/Linux
+    * Append `127.0.0.1       local.exampleapp.com` to the file `/etc/hosts`  
+    * Windows
+        * Append `127.0.0.1       local.exampleapp.com` to the file `C:\Windows\System32\Drivers\etc\hosts`  
+* Follow the README.md inside every sample app, every sample app.
+* Once your app is up and running click the following link to visualize it:
+    * http://local.exampleapp.com:3000
+
+# Environment variables ☀️
+Sample apps use dotenv to load the variables specified in `.env` in your environment.
+
+If you dont want to use dotenv you can set your environment vars manually.
+
+Sample `.env` file
+```
+RSO_BASE_URL=https://auth.riotgames.com
+RSO_CLIENT_ID=riot-example-app
+RSO_CLIENT_SECRET=*******************************
+APP_BASE_URL=http://localhost.riotgames.com:3000
+APP_CALLBACK_PATH=/oauth-callback
+RESPONSE_TYPE=code
+SCOPE=openid
+```
+# Sequence diagram 🟦➡️🟩⬅️
```

Comment:
Comment out all the TDB until they're defined

Reply:
commented, thanks!